### PR TITLE
Overhaul of the CoreOS tutorial (using the new coreOS example in heat…

### PR DESCRIPTION
**Pull request by stefreak**
…templates-examples repository)

@LarsFronius can you review this?

### CODE REVIEW

**Comment by stefreak** [comment]
@LarsFronius  Oh I just noticed i forgot to translate 2 paragraphs. Also I could not test this yet on my machine because of  #24

**Comment by stefreak** [comment]
@LarsFronius I think it should be good now. What do you think?

**Comment by LarsFronius** [comment]
Looks good to me, give me a little time to actually try it out.

**Comment by stefreak** [comment]
No hurries :)

**Comment by stefreak** [comment]
@LarsFronius can you merge this also when you merge this? https://github.com/syseleven/heattemplates-examples/pull/5/files

File: german/docs/tutorials/04-coreos.md [hunk 1]
```diff
@@ -1,86 +1,56 @@
...
-openstack server create --image c4e4ac2c-83db-4331-bf8c-3488aff057c8 --flavor m1.micro --user-data user-data --security-group default --nic 'net-id=aec02c14-5659-4eae-a267-501a28e672b4' --key-name 'mykey' coreos-nginx
+$ openstack stack create -t cluster.yaml <stack name> --parameter key_name=<ssh key name> --wait
 ```
 
-um eine neue Compute Instanz mit CoreOS zu starten, welche den nginx Container starten wird und auf Port 80 erreichbar macht.
+key_name referenziert einen SSH-Key, der wie im [Erste-Schritte-Tutorial](01-firststeps/#ssh-key-hinterlegen) angelegt wurde.
+
+Der Optionale Parameter `number_instances` kann angegeben werden, wenn mehr als eine Instanz hochgefahren werden soll.
+
+## Conclusio: was haben wir erreicht?
```
Comment (gbuerger) [inline german/docs/tutorials/04-coreos.md]: Typo
Instead of "was" it should be "Was"

Comment (stefreak) [inline german/docs/tutorials/04-coreos.md]: fixed :)

**Pull request merged**